sdk/src/Services/DynamoDBv2/Custom/DataModel/IDynamoDBContext.cs

```diff
@@ -170,14 +238,32 @@ public partial interface IDynamoDBContext : IDisposable
 
         #region Transact Get
 
+        /// <summary>
+        /// Creates a strongly-typed TransactGet object, allowing
+        /// a transactional get operation against DynamoDB.
+        /// </summary>
+        /// <typeparam name="T">Type of objects to get.</typeparam>
+        /// <returns>Empty strongly-typed TransactGet object.</returns>
+        TransactGet<T> CreateTransactGet<T>();
+
         /// <summary>
         /// Creates a strongly-typed TransactGet object, allowing
         /// a transactional get operation against DynamoDB.
         /// </summary>
         /// <typeparam name="T">Type of objects to get.</typeparam>
         /// <param name="operationConfig">Config object which can be used to override that table used.</param>
         /// <returns>Empty strongly-typed TransactGet object.</returns>
-        TransactGet<T> CreateTransactGet<T>(DynamoDBOperationConfig operationConfig = null);
+        [Obsolete("Use the CreateTransactGet overload that takes TransactGetConfig instead, since DynamoDBOperationConfig contains properties that are not applicable to BatchGet.")]
```

Comment:
why did you change the signature of the method?

Reply:
See https://github.com/aws/aws-sdk-net/pull/3376#discussion_r1672274431